plugins/woocommerce-admin/client/customize-store/transitional/index.tsx

```diff
@@ -178,23 +178,19 @@ export const Transitional = ( {
 											'woocommerce'
 										) }
 									</p>
-									<Button
-										variant="link"
+									<a
```

Comment:
Let's keep using the `Button` component instead of the "raw" `a` element. It's to keep it up to date with any changes coming from Gutenberg. However `Button` component allows you to pass `href` prop and according to [documentation](https://wordpress.github.io/gutenberg/?path=/docs/components-button--docs):

> `href` - string - If provided, renders `a` instead of `button`.

So something like this should work:

```suggestion
									<Button
										variant="link"
										href={`${ ADMIN_URL }edit.php?post_type=product`}
										onClick={ () => {
											trackEvent(
												'customize_your_store_transitional_product_list_click'
											);
										} }
									>
```

This applies to other links as well.

Reply:
Thanks @kmanijak for the review. I have implemented the feedback, could you please take a look and let me know if any other changes needs to be made.